src/Form/FormAutosuggest.jsx

```diff
@@ -33,7 +33,7 @@ function FormAutosuggest({
   });
   const [isMenuClosed, setIsMenuClosed] = useState(true);
   const [state, setState] = useState({
-    displayValue: '',
+    displayValue: value || '',
```

Comment:
[curious] Is there a test that should be added to account for this change in logic so we don't introduce future regressions? There is a `FormAutosuggest.test.jsx` test suite.

Reply:
@adamstankiewicz yes, you are right. i have added the test

Reply:
@syedsajjadkazmii Thanks for adding the test, LGTM!